scripts/stitching/stitching.py

```diff
@@ -1,24 +1,27 @@
-""" Script for creating a stitched ROI MIBItiff from a folder of FOVs
+""" Script for creating a stitched ROI MIBItiff from a folder of FOVs.
 """
 
-import numpy as np
+import argparse
+import json
 import os
-from mibidata import mibi_image as mi
-from mibitracker.request_helpers import MibiRequests
+import sys
 import time
+
+import numpy as np
+
+from mibidata import mibi_image as mi
 from mibidata import tiff
-import json
+from mibitracker.request_helpers import MibiRequests
```

Comment:
Reordered the imports. Python style is to import 1) standard modules 2) modules installed with conda/pip 3) custom modules, such as those in the repo being worked on